TemporaryOysters/bangazon_website
# Zc product category

## Descriptions:
Created A Page for Individual Product_Types(categories) When a category is clicked on the home page the product category will display along with all products in that Product Type category. 
I also changed a field in the product model on price. To display the price as a number with decimal places instead of a single digit.




## Steps to Test:
git fetch
git branch -r 
git checkout zc-productCategory
update your db run the python server
python manage.py runserver
click on the products to see if the appropriate productType page displays.
